.rubocop.yml

```diff
@@ -349,36 +363,16 @@ RSpec/SubjectStub:
 Rails/FilePath:
   EnforcedStyle: arguments
 
-# Added in v0.81
-Lint/RaiseException:
-  Enabled: true
-
-Lint/StructNewOverride:
-  Enabled: true
-
-Style/HashEachMethods:
-  Enabled: true
-
-Style/HashTransformKeys:
-  Enabled: true
-
-Style/HashTransformValues:
-  Enabled: true
-
-# Added in v0.82
-Layout/SpaceAroundMethodCallOperator:
-  Enabled: true
-
-Style/ExponentialNotation:
-  Enabled: true
-
-# Added in v0.83
-Layout/EmptyLinesAroundAttributeAccessor:
-  Enabled: true
+# This one doesn’t always make sense, it’s mostly the swagger,
+# factories and pact provider states — which need a tear_down block regardless
+Lint/EmptyBlock:
+  Enabled: false
 
-Style/SlicingWithRange:
-  Enabled: true
+Lint/NoReturnInBeginEndBlocks:
+  Exclude:
+    - 'app/services/form1010cg/service.rb'
 
-# Added in v0.84
-Lint/DeprecatedOpenSSLConstant:
-  Enabled: true
+# Will implement this in another PR
+# LOC changed is too high
+Lint/SymbolConversion:
```

Comment:
This cop will be enabled in a separate PR. There were over 1200+ string to symbol conversions and put the LOC changed over 2500+